Vagrantfile

```diff
@@ -120,9 +120,9 @@ make install-libweasyl upgrade-db
 SCRIPT
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "weasyl-debian81.box"
-  config.vm.box_url = "http://deploy.weasyldev.com/weasyl-debian81.box"
-  config.vm.box_download_checksum = "34592e65ebd4753d6f74a54b019e36d1ce006010cb4f03ed8ec131824f45ff9b"
+  config.vm.box = "weasyl-debian91.box"
+  config.vm.box_url = "http://deploy.weasyldev.com/weasyl-debian91.box"
```

Comment:
This redirects to HTTPS; could you update it to specify `https:`?

Reply:
Done!